### PR TITLE
MGMT-3062 Report cluster post-install progress

### PR DESCRIPTION
**Pull request by nmagnezi**

### CODE REVIEW

**Comment by osherdp** [comment]
/uncc @osherdp (as long as it's WIP)

**Comment by nmagnezi** [comment]
/assign @filanov 
can you please review?

File: src/assisted_installer_controller/assisted_installer_controller.go [hunk 1]
```diff
@@ -427,6 +434,50 @@ func (c controller) validateConsolePod() bool {
 	return false
 }
 
+func (c controller) waitForClusterVersion() error {
+	func1 := func() bool {
```
Comment (filanov) [inline src/assisted_installer_controller/assisted_installer_controller.go:438]: `func1`? maybe `getClusterVersionAndUpdateProgress`?

Comment (nmagnezi) [reply src/assisted_installer_controller/assisted_installer_controller.go:438]: np. will change that.

Comment (nmagnezi) [reply src/assisted_installer_controller/assisted_installer_controller.go]: Done

**Pull request merged**